build.gradle

```diff
@@ -17,11 +18,30 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'org.springframework.security:spring-security-test'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
-    testImplementation 'org.hamcrest:hamcrest-library:2.2'
+    testImplementation 'io.rest-assured:rest-assured:4.4.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
 }
 
 test {
-	useJUnitPlatform()
+	useJUnitPlatform{
+	    includeTags 'unit'
+	}
+    testLogging {
+        events "passed", "skipped", "failed"
+    }
 }
+
+task integrationTest(type: Test) {
+    dependsOn build
```

Comment:
this i do not like, but it breaks otherwise when build dir is clean :S

Reply:
it's fine. Gradle won't build it from scratch if it already built. It will use cache